docker/cygnus-ngsi/README.md

```diff
@@ -1,194 +1,3 @@
-time=2016-05-05T10:01:32.051UTC | lvl=INFO | corr=8bed4f8d-c47f-499a-a70d-365883584ac7 | trans=8bed4f8d-c47f-499a-a70d-365883584ac7 | srv=default | subsrv=/ | function=processNewBatches | comp=cygnus-ngsi | msg=com.telefonica.iot.cygnus.sinks.NGSISink[423] : TTL exhausted, finishing internal transaction (8bed4f8d-c47f-499a-a70d-365883584ac7)
-```
-
-Don't worry about the error, it is normal (please, see next section).
-
-You can stop the container as:
-
-```
-$ docker stop 9ce0f09f5676
-9ce0f09f5676
-$ docker ps
-CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
-```
-
-[Top](#top)
-
-###<a name="section3.2"></a>Using a specific configuration
-As seen above, the default configuration distributed with the image is tied to certain values that may not be suitable for you tests. Specifically:
-
-* It only works for building historical context data in MySQL.
-* The endpoint for MySQL is `mysql`.
-* The logging level is `INFO`.
-* The logging appender is `console`.
-
-You may need to alter the above values with values of your own.
-
-[Top](#top)
-
-####<a name="section3.2.1"></a>Editing the docker files
-The easiest way is by editing both the `Dockerfile` and/or `agent.conf` file under `docker/cygnus-ngsi` and building the cygnus-ngsi image from the scratch.
-
-This gives you total control on the docker image.
-
-[Top](#top)
-
-####<a name="section3.2.2"></a>Environment variables
-Those parameters associated to an environment variable can be easily overwritten in the command line using the `-e` option. For instance, if you want to change the log4j logging level, simply run:
-
-    $ docker run -e CYGNUS_LOG_LEVEL='DEBUG' cygnus-ngsi
-    
-Or if you want to configure non empty MySQL user and password:
-
-    $ docker run -e CYGNUS_MYSQL_USER='myuser' -e CYGNUS_MYSQL_PASS='mypass' cygnus-ngsi
-
-[Top](#top)
-
-####<a name="section3.2.3"></a>Using volumes
-Another possibility is to start a container with a volume (`-v` option) and map the entire configuration file within the container with a local version of the file:
-
-    $ docker run -v /absolute/path/to/local/agent.conf:/opt/apache-flume/conf/agent.conf cygnus-ngsi
-
-Of course, you can combine volumes and environment variables overwriting:
-
-    $ docker run -v /absolute/path/to/local/agent.conf:/opt/apache-flume/conf/agent.conf -e LOG_LEVEL='DEBUG' cygnus-ngsi
-
-[Top](#top)
+Please, refer to [root docker documentation](../../../doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md) if you want to use a docker image for cygnus-ngsi.
```

Comment:
This about root... what does it means in this context? I think something like "Please, refer to the documentation if you want..."